plone/app/vocabularies/users.py

```diff
@@ -153,7 +153,7 @@ class UsersSourceQueryView:
     template = ViewPageTemplateFile("searchabletextsource.pt")
 
     def __init__(self, context, request):
-        msg = "UsersSourceQueryView is deprecated and will be removed on " "Plone 6"
+        msg = "UsersSourceQueryView is deprecated and will be removed on Plone 7"
```

Comment:
This was added, funnily enough by me, back in 2016, do we really have to bump the deprecation notice for yet another major version? 😅 

Interestingly enough, we are not warning via `zope.deprecation` but by using python stdlib `warnings.warn`. Is there any preference of one or the other? Could we, pun intended, deprecate `zope.deprecation` and use plain stdlib `warnings.warn`? 🤔